brainstormforce/ultimate-addons-for-gutenberg
# Fix: Icon list - Layout issue when the user close Preset tab.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/560b03e7-3835-4fd6-8994-487ca07a1b4a?redirect=true&view=06c01065-22db-4b44-9a83-f0eceb06e66a&t=attributes

### Screenshots

- https://share.getcloudapp.com/lluEW5xX

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop Icon List.
- Add layout H/V any. Close the preset tab and check is it changing. (It should not change)
- Please check ss.

### Checklist:
